src/types.ts

```diff
@@ -101,3 +101,32 @@ export interface Column {
   stylingInfo: string[];
   sortDirection: string;
 }
+
+export interface PageInfo {
+  page: number;
+  rowsPerPage: number;
+  rowsPerPageOptions: number[];
+}
+
+export type SetPageInfo = stardust.SetStateFn<{
+  page: number;
+  rowsPerPage: number;
+  rowsPerPageOptions: number[];
+}>;
+
+export type Announce = ((arg0: AnnounceArgs) => void) | undefined;
+
+export interface RenderProps {
+  direction: 'ltr' | 'rtl';
+  selectionsAPI: stardust.ObjectSelections;
+  rootElement: HTMLElement;
+  tableData: object;
+  pageInfo: PageInfo;
+  setPageInfo: SetPageInfo;
+  constraints: stardust.Constraints;
+  translator: ExtendedTranslator;
+  theme: ExtendedTheme;
+  keyboard: stardust.Keyboard;
+  footerContainer: HTMLElement;
+  announce: (...args: any[]) => any;
```

Comment:
should be more strict, should be a type for it already

Reply:
I tried with `announce: Announce` which does not work. But I gonna look into it again.

Reply:
Will get it done when working on TableWrapper.jsx

Reply:
Done, may need to do some change working on TableWrapper.jsx